user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/dedicated_clustered_storage/Configuring_database/Configuring_multiple_Elasticsearch_clusters.md

```diff
@@ -36,8 +65,6 @@ To configure this setup:
 
    - *FileOffloadIdentifier*: String used to identify this connection. Each connection should have a different identifier, which will be used for file offloads.
 
-   - *ID*: From DataMiner 10.4.0/10.3.10 onwards, this attribute is added by configuring EMCO in cube, it is used to pair clusters with their encrypted passwords *Do not change this field manually*.
-
```

Comment:
@MichielMN, I believe this is not necessary information if, from 10.3.10/10.4.0 onwards, users no longer need to access the *DBConfiguration.xml* file to configure the Elasticsearch clusters. If this is important information, it should be incorporated in the new procedure.

Reply:
Also not sure what to do with this, people aren't supposed to edit that file manually anymore but they still will, I added this to make it known that it is imperative that that particular field isn't edited or they'll have to redo the entire config